src/components/AddPlaidBankAccount.js

```diff
@@ -101,8 +100,9 @@ class AddPlaidBankAccount extends React.Component {
             institution: {},
         };
 
-        this.getErrors = () => ReimbursementAccountUtils.getErrors(this.props);
-        this.clearError = inputKey => ReimbursementAccountUtils.clearError(this.props, inputKey);
+        this.clearError = ReimbursementAccountUtils.clearError.bind(this);
+        this.getErrors = ReimbursementAccountUtils.getErrors.bind(this);
+        this.setErrors = ReimbursementAccountUtils.setErrors.bind(this);
```

Comment:
NAB, personally, I think it's fine to explicitly pass the props rather than implicitly bind to `this`. Perhaps it would be better to not spend time improving things we know we are going to soon replace and only change what we need to accomplish the task at hand.

Reply:
Agreed, but I changed this because I started using `this.state` and `this.setState` in `clearError`. Didn't check this properly, but I thought that passing `this.setState` would already require a binding so I thought it was just better to bind `clearError`. (https://github.com/Expensify/App/pull/6200#discussion_r757510818)